DESIGN.md

```diff
@@ -6,47 +6,63 @@ Encrypted protocol
 
 The client and server share a key file with 256 or more bits of entropy.  On
 launch, they read the key file and compute
+
     K = SHA256(key file).
 
 When a connection is established:
-C1. The client generates a 256-bit random value nonce_C and sends it.
```

Comment:
Not certain if you want to leave DESIGN in plain-text, or markdownify it.  I definitely think that README should be markdown (to display nicely on github), but there's an argument in favor of not changing DESIGN since it's only aimed at cryptographers (who will surely be familiar with plaintext math).  I can remove that change easily if you want.

I also noticed that BUILDING is plaintext; I'd be happy to markdownify that as well.  Also, BUILDING isn't linked (in terms of href) from README.md... basically, my rule of thumb was that if something is linked directly from README.md, it should look nice in the github interface, which means markdown or extremely simple plain text (which DESIGN and BUILDING do not qualify as).